File: system_template/test/2_namespaces/data/system_template/on_conflict/_map.py
[
    *[{
        "source": "on_conflict.txt",
        "on_conflict": "append_start",
        "target": "BP/on_conflict.txt"
    } for _ in range(10)],
    *[{
        "source": "on_conflict.mcfunction",
        "on_conflict": "append_end",
        "target": AUTO_FLAT
    } for _ in range(10)],
    *[{
        "source": "on_conflict.behavior.json",
        "on_conflict": "merge",
        "json_template": True,
        "scope": {
            "event": f"my_event_{i}"
        },
        "target": AUTO_FLAT
    } for i in range(10)]
]

File: modular_mc/test/packs/data/modular_mc/ordered_entries/ordered_entries_3/_map.ts
export const MAP = [
	{
		source: "ordered_entry.behavior.json",
		target: ":autoFlat",
		jsonTemplate: true,
		onConflict: "merge",
		executionOrder: 3,
		scope: {
			input: "ENTRY 3",
		},
	},
];
